commands/auth.go

```diff
@@ -145,7 +144,7 @@ func loginCmdF(cmd *cobra.Command, args []string) error {
 	}
 
 	if accessToken != "" && username != "" {
-		return errors.New("You must use --access-token or --username, but not both.")
+		return fmt.Errorf("you must use --access-token or --username, but not both")
```

Comment:
Could you please elaborate a bit why do you want to change this? `fmt.Errorf` is not _needed_ here, so why use it?

The advantage of using `errors.New` is, that you can easily see, if a Formatierung string is used or not.

https://github.com/pkg/errors has both a `New` and a `Errorf` method :wink:

Reply:
To be more consistent actually (a little syntactic sugar). Since you can use `fmt.Errorf` for both cases, it reduces the confusion between `fmt.Errorf` and `errors.New`. Also one less `import` statement 😇

Reply:
The `import` statement isn't a selling point for me, as 99% of the time the editor just take care of this. What is the confusion you pointed out? `fmt.Errorf` and `errors.New` are doing _different_ things and hence should be distinct.

Reply:
By confusion I mean easily seeing the errors. It's not a big deal though. Also, as we discussed in the previous comment, if we use `pkg/errors` and since it has both methods and It will remove the `errors` and `fmt` distinction.